pom.xml

```diff
@@ -584,41 +613,44 @@
         <guava.version.range>[18.0,19.0)</guava.version.range>
         <gson.version>2.6.2</gson.version>
         <gson.version.range>[2.6.2,3)</gson.version.range>
-        <orbit.org.yaml.version>1.16.0.wso2v1</orbit.org.yaml.version>
-        <orbit.org.yaml.version.range>[1.16.0, 1.17.0)</orbit.org.yaml.version.range>
+        <org.snakeyaml.version>1.17</org.snakeyaml.version>
```

Comment:
With the kernel 5.2.0, we are using OSGified YAML bundle. So we don't need to have Orbit bundle for YAML